README.md

```diff
@@ -311,12 +313,15 @@ Sentry.
 - **context:** Optional dictionary. See "context" note below.
 
 ### ProxyServer
-- **routes:** Required list of one or more ProxyRoute dictionaries
+- **routes:** Optional list of one or more ProxyRoute dictionaries.
```

Comment:
I added a note about when to use each.

This changeset had already added TCP (and also HTTP) balancing to the features list at the top of the README.md - were you suggesting that I add it somewhere else?